dataverse/orgsvc/C#/ActivityPartyRecords/README.md

```diff
@@ -1,30 +1,28 @@
 
 # Work with activity party records
 
-This sample code shows how to work with activity party records.
+This sample creates some sample data to show you how to work with [activity party](https://learn.microsoft.com/power-apps/developer/data-platform/activityparty-entity?view=dataverse-latest) records.
 
-## How to run this sample
-
-See [How to run samples](https://github.com/microsoft/PowerApps-Samples/blob/master/dataverse/README.md) for information about how to run this sample.
+For more information, see [ActivityParty entity](https://learn.microsoft.com/previous-versions/dynamicscrm-2013/developers-guide/gg328549(v=crm.6)).
```

Comment:
I would not link to a 2013 version of our docs. That same article is available here: [ActivityParty entity](https://learn.microsoft.com/dynamics365/customerengagement/on-premises/developer/activityparty-entity?view=op-9-1)

Reply:
agree

But please refer to this https://learn.microsoft.com/power-apps/developer/data-platform/activityparty-entity